draft-ietf-quic-transport.md

```diff
@@ -3724,6 +3724,10 @@ PROTOCOL_VIOLATION (0xA):
 : An endpoint detected an error with protocol compliance that was not covered by
   more specific error codes.
 
+TLS_NEGOTIATION_FAILURE (0xB):
```

Comment:
You need to fix `UNSOLICITED_PATH_RESPONSE (0xB)` below so there isn't collision on the error codes. That being said, what's wrong with `TLS_HANDSHAKE_FAILED` defined in the QUIC TLS spec?

Reply:
Nothing wrong with TLS_HANDSHAKE_FAILED. Apart from not being documented in the transport draft... I updated the PR to just a pointer to the TLS draft.